packages/component-library/src/utils/formatters.js

```diff
@@ -0,0 +1,18 @@
+import { format } from 'd3-format';
+
+export const numeric = (d) => {
+  let formatted;
+
+  if (d >= 1000000) {
+    formatted = format('.2s')(d)
+      .replace('M', ' million')
```

Comment:
Any ideas on how I could make this less rote?

Reply:
A couple other caveats here: this won't work for billions, which uses `G` since it's scientific notation. It also won't work for trillions (which we could run into if we do budget projects in the future).

Given all these caveats, it may make more sense to write our own formatter rather than hacking on top of d3's formatter in this case.